ReactWindows/ReactNative.Shared/UIManager/UIViewOperationQueue.cs

```diff
@@ -480,19 +481,16 @@ private void OnRendering(object sender, object e)
                 }
             }
 
+            Action[] actions = EMPTY_ACTIONS_ARRAY;
             lock (_gate)
             {
-                try
-                {
-                    foreach (var batch in _batches)
-                    {
-                        batch();
-                    }
-                }
-                finally
-                {
-                    _batches.Clear();
-                }
+                actions = _batches.ToArray();
```

Comment:
I'd be concerned that because this is happening at ~60fps, without a pooling data structure to clone into, we'd put too much pressure on the GC.

---
In reply to: [88969471](https://github.com/ReactWindows/react-native-windows/pull/891#discussion_r88969471) [](ancestors = 88969471,88968974)